python/ray/_private/accelerators/amd_gpu.py

```diff
@@ -32,6 +32,12 @@ def get_resource_name() -> str:
 
     @staticmethod
     def get_visible_accelerator_ids_env_var() -> str:
+        global HIP_VISIBLE_DEVICES_ENV_VAR
+        hip_val = os.environ.get(HIP_VISIBLE_DEVICES_ENV_VAR, None)
+        cuda_val = os.environ.get(CUDA_VISIBLE_DEVICES_ENV_VAR, None)
+        if hip_val is None and cuda_val is not None:
+            HIP_VISIBLE_DEVICES_ENV_VAR = CUDA_VISIBLE_DEVICES_ENV_VAR
```

Comment:
I don't think we should change the value of `HIP_VISIBLE_DEVICES_ENV_VAR`. It should be treated as a constant. Instead can we just 

```
@staticmethod
    def get_visible_accelerator_ids_env_var() -> str:
        hip_val = os.environ.get(HIP_VISIBLE_DEVICES_ENV_VAR, None)
        cuda_val = os.environ.get(CUDA_VISIBLE_DEVICES_ENV_VAR, None)
        if hip_val is None and cuda_val is not None:
            return CUDA_VISIBLE_DEVICES_ENV_VAR
        else:
            return HIP_VISIBLE_DEVICES_ENV_VAR
```